tests/test_approval_modules_snapshots/cdf_apm_simple.yaml

```diff
@@ -326,15 +326,6 @@ DataModel:
     space: apm_simple
     type: view
     version: '1'
-Database:
-- name: apm_simple
-- name: apm_simple
-- name: apm_simple
-- name: apm_simple
-- name: apm_simple
-- name: apm_simple
-- name: apm_simple
-- name: apm_simple
```

Comment:
Database is no longer explicitly created. Instead it is included when uploading the raw tables with the ensure_parent arg.